src/applications/static-pages/mhv-signin-cta/components/messages/CustomAlert.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+// eslint-disable-next-line import/no-named-default
+import { default as recordEventFn } from '~/platform/monitoring/record-event';
+import classNames from 'classnames';
+
+/**
+ * Custom alert similar to VAAlert that allows to set the icon.
+ * @property {*} children the children components to include inside the alert
+ * @property {bool} disableAnalitics true to disable analytics
+ * @property {string} headline the text for the headline of the alert
+ * @property {*} recordEvent the function to record the event
+ * @property {string} status the status of the alert ('info', 'success', 'warning' or 'continue')
+ * @property {string} icon the name of the icon for the alert
+ */
+const CustomAlert = ({
```

Comment:
Note: This custom alert is based on the custom `src/applications/mhv-landing-page/components/MhvRegistrationAlert.jsx`. We need to refactor this file out to a common location and use it for our other alerts once this PR is approved.